Main.py

```diff
@@ -1,26 +1,30 @@
-from Exam_question_class import ExamQuestion
 import Exam_class
 from Student_class import Student
-from Discipline_class import Discipline
 from Adress_class import Adress
 import sec_class
 from Teacher_class import Teacher
-import Teacher_class
 from Director_class import Director
 from School_class import School
 from classes_class import Classes 
 from class_list import SchoolClassList
-from info_class import Info
 from Math_class import Math
 from English_class import English
 from Physics_class import Physics
-from new_student1_parent_class import Student1ExamPoints
-from new_student2_parent_class import Student2ExamPoints
+from Parent1_read_points import Student1ExamPoints
```

Comment:
Having something which counts in a class or function name always feels weird (and can be a code smell). Maybe we should talk this can be solved differently.